app/views/reports-list/ap-listing.njk

```diff
@@ -11,21 +12,39 @@
       text: "Report List",
       href: "/report-list"
     },{
-      text: "AP Listing Report",
+      text: "AP/AR Listing Report",
       href: "/report-list/ap-listing"
```

Comment:
likewise I'd mirror this href in line with the generic term we use in report types

Reply:
changed naming to be ap-ar-listing